app/views/devise/sessions/new.html.erb

```diff
@@ -1,26 +1,65 @@
-<h2>Log in</h2>
+<%
+=begin %>
```

Comment:
What's this for?

Reply:
Oh, I was saving the original code while I was trying out the bootstrap styling in case if I messed up. Let me go ahead and erase that.